memory/src/columns.rs

```diff
@@ -30,7 +29,9 @@ pub struct MemoryCols<T> {
     /// A boolean flag indicating whether addr' - addr == 0
     pub addr_not_equal: T,
 
+    /// Counter and lookup multiplicities for local range check
```

Comment:
Just a tiny nit, but I think doc comments should ideally be about a single field given how IDEs etc. interpret them